org.eclipse.languageserver/src/org/eclipse/languageserver/LSPEclipseUtils.java

```diff
@@ -42,8 +42,8 @@ public static int toOffset(Position position, IDocument document) throws BadLoca
 		return document.getLineInformation(position.getLine()).getOffset() + position.getCharacter();
 	}
 
-	public static TextDocumentPositionParamsImpl toTextDocumentPosistionParams(URI fileUri, int offset, IDocument document)
-			throws BadLocationException {
+	public static TextDocumentPositionParamsImpl toTextDocumentPosistionParams(URI fileUri, int offset,
+	        IDocument document) throws BadLocationException {
```

Comment:
Do you like this kind of things? I'm not convinced about the benefit of limiting line width if it ends with many lines splitted.